flake.nix

```diff
@@ -1,6 +1,6 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/master";
```

Comment:
With `nixos-unstable` we need to wait a few days to be able to use the PRs that we get merged into nixpkgs. This is annoying.

The downside of using `master` is that if when we update flake.lock, some packages that we need are not yet in cache.nixos.org, and we have to build them. But this is only once (as they are then in our cachix) and also if the commit is at least a few hours old, cache.nixos.org is already mostly primed.

Reply:
Building package is not an issue as long it does not have bugs :)